flytekit/core/promise.py

```diff
@@ -332,6 +332,14 @@ def __init__(self, var: str, val: Union[NodeOutput, _literal_models.Literal]):
     def __hash__(self):
         return hash(id(self))
 
+    def __rshift__(self, other: typing.Union[Promise, VoidPromise]):
```

Comment:
Not for this PR, but we should support chaining of multiple tasks/nodes, for example, this should work:
```
t1 >> t2 >> t3
```
but right now we only have support for pair-wise constraints, in other words, we have to specify separately:
```
t1 >> t2
t2 >> t3
```

Reply:
This can get pretty crazy, imagine someone doing:
```
t1 >> t2 << t3
```
:joy: